README.md

```diff
@@ -53,7 +53,7 @@ quickly get libdragon up and running. Basically:
 2. Install the [the libdragon CLI](https://github.com/anacierdem/libdragon-docker).
    You have two options:
 
-   1. Download the [pre-built binary](https://github.com/anacierdem/libdragon-docker/releases/tag/v10.8.0), 
+   1. Download the [pre-built binary](https://github.com/anacierdem/libdragon-docker/releases/tag/v10.9.0), 
```

Comment:
Instead we can use the latest tag: `https://github.com/anacierdem/libdragon-docker/releases/latest`

Reply:
Okay. Addressed in new commit.